### ISSUE TITLE
relation between item name, name statement and labels?

### ISSUE THREAD

**Issue by joelthill** [weso/SAF-Lux#28]
what is the relation between the 3+ occurences of "name" : item name (next to the Q-id), name statement (at the bottom) and labels in different languages? how are they loaded? how do they behave if any of them is modified?

why is the name statement at the very bottom of the statements (looks like the most important one)? can they be sorted differently?

**Comment by mzeinstra** [comment]
@joelthill it would be good to add a screenshot of what you are seeing to help us document an reply to this.

**Comment by joelthill** [comment]
![Screenshot_2021-07-14 Zenner, Theodor(1)](https://user-images.githubusercontent.com/10212884/125594457-f714ce04-8093-4c81-8979-0ddcb4870c2b.png)
![Screenshot_2021-07-14 Zenner, Theodor](https://user-images.githubusercontent.com/10212884/125594460-76647028-8352-485f-b2d7-89c31879a51b.png)

**Comment by digitaalwerktuig** [comment]
I expect that after initial (bulk) import, they have to be maintained individually? Is this correct @andrawaag?

**Comment by mzeinstra** [comment]
They are currently indeed replicated from the heading:name column of your test data.

But I think your issues is not the duplication, rather the sorting of statements?

**Comment by joelthill** [comment]
replication is an issue. dont' you agree that expecting manual handling of coherency is doomed from the start?
sorting is more like a cosmetic thing - even though I remember that some collegues from the business saw this differently when we first adressed this subject.

**Comment by digitaalwerktuig** [comment]
I see your point regarding the replication and conseqent manual management of the coherency. But since I was working on an assuption in my response, it would be great if @andrawaag can also provide his response.

**Comment by andrawaag** [comment]
This is inherent to the Wikibase model. A Wikibase item consists of a header, set of statements and footer. In this example, the name is part of the header and the set of statements. At the header, it is not possible to add the provenance of the names, nor easily align with CIDOC-CRM. That is why it is replicated in the statements to both provide its provenance and mapping to CIDOC-CRM. 
The header is more a label for quick reference but the structured part is captured in the statement section of a wikibase item. 

Coherency between the statements and labels can be maintained if edits are only done at the statement level, while the replication is handled by scripts. 
Coincidentally, I handled a similar issue on Wikidata a month ago, where species common names are stored in Wikidata on the statements level, but not in the headers. I wrote the following prototype to update the headers based on the statements that captured referenced common names: https://public.paws.wmcloud.org/User:Andrawaag/wikiproject_biodiversity/commonlabelsasaliases.ipynb. 

For the names to be captured as structured data with references they need to be in the statements. For the names to be captured in search indexes (i.e. to be found through the search fields) they need to be in the headers.
